internal/provider/data_source_aci_endpoint_security_group_test.go

```diff
@@ -14,15 +14,14 @@ import (
 func TestAccDataSourceFvESgWithFvAp(t *testing.T) {
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { testAccPreCheck(t) },
+		PreCheck:                 func() { testAccPreCheck(t, "apic", "5.0(1k)-") },
 		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
 		Steps: []resource.TestStep{
 			{
 				Config:             testConfigFvESgDataSourceDependencyWithFvAp,
 				ExpectNonEmptyPlan: false,
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckResourceAttr("data.aci_endpoint_security_group.test", "name", "test_name"),
-					resource.TestCheckResourceAttr("data.aci_endpoint_security_group.test", "admin_state", "no"),
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.